### PR TITLE
URL.createObjectURL should not work with MediaStream

### PR DESCRIPTION
**Pull request by annevk**
Change the one existing test to no longer rely on it (and clearly mark it as manual) and add an entry to historical.html for it.

### CODE REVIEW

**Comment by annevk** [comment]
Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1454889.

File: mediacapture-streams/historical.html [hunk 1]
```diff
@@ -15,4 +15,8 @@
 test(function() {
   assert_false("mozGetUserMedia" in navigator);
 }, "navigator.mozGetUserMedia should not exist");
+
+test(() => {
+  assert_throws(new TypeError(), () => URL.createObjectURL(new MediaStream()));
```
Comment (Ms2ger) [inline mediacapture-streams/historical.html]: Can you move the creation of the `MediaStream` object out of the `assert_throws`?

**Review by foolip — APPROVED** [review]

File: mediacapture-streams/historical.html (hunk 1 above)
Comment (foolip) [reply mediacapture-streams/historical.html]: Yeah, that'd be good.

**Comment by foolip** [comment]
@Orphis this is why I poked you about finishing https://crbug.com/800767 :)

**Comment by annevk** [comment]
Filed https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/17021183/ on Edge so all failing browsers have a bug now.

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**